docs/examples/frameworks/pytorch/pytorch-external_input.ipynb

```diff
@@ -24,8 +24,9 @@
     "import numpy as np\n",
```

Comment:
<p>I think you can jus remove size or set it to -1 so <code>StopIteration</code>  will just stop the PyTorchIterator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
When I removed size the iterator ran indefinitely in epoch 0.

Reply:
<p>mhhh, strange</p>